fastlane/lib/fastlane/actions/docs/sync_code_signing.md

```diff
@@ -509,12 +509,16 @@ Please be careful when using this option and ensure the certificates and profile
 
 ### Manual Decrypt
 
-If you want to manually decrypt a file you can.
+If you want to manually decrypt or encrypt a file, you can use the companion script `match_file`:
 
 ```no-highlight
-openssl aes-256-cbc -k "<password>" -in "<fileYouWantToDecryptPath>" -out "<decryptedFilePath>" -a -d -md [md5|sha256]
+match_file encrypt "<fileYouWantToEncryptPath>" ["<encryptedFilePath>]"
+
+match_file decrypt "<fileYouWantToDecryptPath>" ["<decryptedFilePath>]"
```

Comment:
I think that once you install the gem, everything under `bin` will be in your CLASSPATH. SO `match_file` should be enough.

The [] were because the second parameter is optional and match_file decrypts / encrypts in place when omitted.